specification/web/resource-manager/Microsoft.Web/stable/2022-09-01/StaticSites.json

```diff
@@ -3811,6 +4562,130 @@
         }
       }
     },
+    "StaticSiteDatabaseConnectionConfigurationFileOverview": {
+      "description": "A database connection configuration file",
+      "type": "object",
+      "properties": {
+        "fileName": {
+          "description": "The name of the configuration file.",
+          "type": "string",
+          "readOnly": true
+        },
+        "contents": {
+          "description": "The Base64 encoding of the file contents.",
+          "type": "string",
+          "readOnly": true
+        },
+        "type": {
+          "description": "The type of configuration file.",
+          "type": "string",
+          "readOnly": true
+        }
+      },
+      "x-ms-client-flatten": true
+    },
+    "DatabaseConnection": {
+      "description": "Static Site Database Connection ARM resource.",
```

Comment:
>ARM [](http://example.com/codeflow?start=54&length=3)

Think ARM as implementation detail... instead of an ARM resource, better to call it a resource, or an Azure resource, or even just say its "A static site database connection [that you can use to ...]" .